jina/executors/frameworks.py

```diff
@@ -14,7 +14,9 @@ class BaseFrameworkExecutor(BaseExecutor):
 
     def __init__(self, model_name: str = None, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        import GPUtil
         self.model_name = model_name
+        self.on_gpu = True if self.on_gpu and GPUtil.getGPUs() else False
```

Comment:
again, redundant boolean logic
```suggestion
        self.on_gpu = self.on_gpu and GPUtil.getGPUs()
```